lms/djangoapps/courseware/views.py

```diff
@@ -862,6 +876,59 @@ def get_cosmetic_display_price(course, registration_price):
         return _('Free')
 
 
+@require_global_staff
+@ensure_valid_course_key
+def enroll_staff(request, course_id):
```

Comment:
A better approach would have been to implement django View with GET and POST methods